docs/TrafficSplitting.md

```diff
@@ -0,0 +1,166 @@
+# Traffic Splitting
+
+kn can help you control which revisions get traffic in your Knative Service.
```

Comment:
This sentence feels odd as it stands by itself. Maybe add TL;DR or Summary or something.

Reply:
I re-wrote this after earlier review comments from @sixolet and others. I think the gist is upto the point about what feature we are to detail below.